Formula/t-rec.rb

```diff
@@ -0,0 +1,23 @@
+class TRec < Formula
+  desc "Blazingly fast terminal recorder that generates animated gif images for the web"
+  homepage "https://github.com/sassman/t-rec-rs"
+  url "https://github.com/sassman/t-rec-rs/archive/v0.2.1.tar.gz"
+  sha256 "a23ae0e19b76740220f4640a82d2813fe1002609bbf28ab180f3fc9228b141e3"
+  license "GPL-3.0-only"
+
+  depends_on "rust" => :build
+  depends_on "imagemagick"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    # let's fetch the window id
+    o = `#{bin}/t-rec -l | tail -1`
+    win_id = o.split(/\s|\n/)[-1]
+    # verify that it's an appropriate id
+    assert_equal win_id && Integer(win_id).positive?, true
+    # unfortunately grabbing a window screenshot does not work on github workers, so we cannot test this here
```

Comment:
Is there no command where you can actually check something about the output? This doesn't really say anything about the functionality of the program

Reply:
Well as you can see in line 18 I split the output and verify that the second portion of a string in the format of `something | number` is a number. Furthermore, I'd like to point you to the [comment above](#issuecomment-730688428) where I'm trying to explain why a full featured test does not work on github actions, locally however it does work. The nature of a headless build server seems to be my guess of the culprit.

However, if you have specific suggestions feel free to propose them, I'm happy to implement them.

Reply:
What if you just pick an impossibly high window number and try and run the program on that? I expect it'll return some kind of error and you can check if the error code and text is as expected.

Reply:
Sure, I can test that as well.

Reply:
done